.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+
+services:
+  - docker
+
+go:
+  - 1.7
```

Comment:
Oh, and one important thing with gimme: if you specify only 1.7, it's going to install 1.7, not the latest 1.7 release, so you probably wanna use ``1.7.4`` to make sure you have latest security fixes.